scripts/testing/setup_localnet_nodes.sh

```diff
@@ -434,53 +434,53 @@ required_variables=(
     "UPGRADE_HEIGHT"
 )
 
-# Check if screen is installed
-check_screen_installed
+# # Check if screen is installed
+# check_screen_installed
```

Comment:
Is there any specific reason commenting this out? Probably remove if not required?

Reply:
@kenta-elys yep I think you need to revert those changes from this file I will fix the lz4 and sed issues in a later PR

Reply:
@cosmic-vagabond Done ,reverted.